src/main/java/helpers/DataHelper.java

```diff
@@ -0,0 +1,51 @@
+package helpers;
+
+import com.github.javafaker.Faker;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Random;
+
+public class DataHelper {
+    private static Faker faker = new Faker();
+    private static Random random = new Random();
+
+    public static String getRandomEmail() {
+        return faker.internet().emailAddress();
+    }
+
+    public static String getRandomText(int number) {
+        String result = "";
+        for(int i = 1;i <= number; i++){
+            result += "?";
```

Comment:
you could use this to repeat a character: http://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/StringUtils.html#repeat(java.lang.String,%20int)

Reply:
Thank you. I replaced it.